docs/wiki/modules/assessment.md

```
@@ -0,0 +1,24 @@
+# Module: Assessment
+
+Batch assessments of datasets against a config (prompt_template + text columns + attachments over rows). No deep-dive doc yet.
+
+All paths relative to `backend/app/`.
+
+## Routes
+- `api/routes/assessment/`
+
+## Tables (SQLModel)
+| Table | Model |
```

markdownlint-cli2 (0.22.1)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)